posts/martinfernandezcx/jwt-nginx-auth.md

```
@@ -0,0 +1,274 @@
+   javascript
+   pm.test("Status code is 200", function () {
+       pm.response.to.have.status(200);
+   });
+   pm.test("Response has token", function () {
+       var jsonData = pm.response.json();
+       pm.expect(jsonData).to.have.property('token');
+   });
+   ```
+
+2. **Protected Route Test**
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)